mina/middleware/src/endpoints/write.ts

```diff
@@ -0,0 +1,20 @@
+import type { ActionResult } from '../types';
+import { builder } from '@paima/sdk/concise';
+import { getDefaultActiveAddress, postConciselyEncodedData } from '@paima/sdk/mw-core';
+import { ENV } from '@paima/sdk/utils';
+
+async function gainExperience(count: number): Promise<ActionResult> {
+  const userWalletAddress = getDefaultActiveAddress();
```

Comment:
is this actually used? We have a bunch of stuff for XP / gainExperience, but I'm not sure this is actually used in the actual template

Reply:
It's leftover copypasta from the generic template. I can rip it out but it would leave the `api` and `game-logic` subprojects virtually empty.